index.js

```diff
@@ -301,38 +278,12 @@ class PluginXrpPaychan extends PluginBtp {
       return
     }
 
-    const tx = await this._api.preparePaymentChannelClaim(this._address, {
+    return this._txSubmitter('preparePaymentChannelClaim', {
```

Comment:
I think this should just be awaited instead of returned, because we don't want to do anything with the return value